CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [1.32.1](https://github.com/brave/brave-ios/releases/tag/v1.32.1)
+
+- Added CarPlay support for playlist. ([#3673](https://github.com/brave/brave-ios/3673))
+- Added a new section in history for entries older than last month. ([#4040](https://github.com/brave/brave-ios/4040))
+- Added support for sharing playlist items. ([#4191](https://github.com/brave/brave-ios/4191))
+- Added an option to disable "Add to playlist" button in toolbar.  ([#4234](https://github.com/brave/brave-ios/4234))
+- Implemented 'Bookmark all tabs' feature. ([#4132](https://github.com/brave/brave-ios/4132))
```

Comment:
nit: use `"` or the CHANGELOG will get formatted. We usually don't use the `formatting` but use "formatting".